src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -648,16 +648,18 @@ describe('ReactUpdates', () => {
           'Inner-render-1-0',
           'Inner-didUpdate-1-0',
         'Outer-didUpdate-1',
+           // Happens in a batch, so don't re-render yet
           'Inner-setState-1',
-            'Inner-render-1-1',
```

Comment:
We should figure out why there was a change before we can make a meaningful decision on that point. We've made other breaking changes for Fiber so we'll need to do a 16 regardless before releasing Fiber (or maybe at the same time).